source/index.html.md

```diff
@@ -1,239 +1,674 @@
+
+## Testing Authentication
+
+Here's how to verify your API Key:
+
+###<div class="colour-pill"><span class="get">GET</span> Verify API Key</div>
+
+Ping the API at `http://api.namara.io/v0/users/verify_token` to ensure that the API token is valid for the user. You don't have to pass `api_key` into the URL - you can pass it in through the headers.
+
+>1)
+`{ "status": true }`
+
+>2)
+`{ "status": false }`
+
+Response | Description
+-------- | -----------
+200: OK | The API Key is valid (*example 1*)
+422: Unprocessable Entity | API does not correspond to any user (*example 2*)
+
+# Data API
+
+> https://:namara_api_host/v0/data_sets/:data_set_id/data/:version
+
+Here, we'll outline the specifications for accessing the Data API. Each data set can be accessed at the API URL. In Namara, click on the "API Info" tab when viewing a data set to see all information regarding ID, version, and properties.
+
+##<div class="colour-pill"><span class="get">GET</span> Data Query</div>
+
+> https://api.namara.io/v0/data_sets/:data_set_id/data/:version
+
+A useful feature is the ability to query a data set by passing in parameters. The query parameters are optional; `data_set_id` and `version` are *path parameters* and **must** be in the call. Here are the ways in which you can manipulate the response:
+
+Parameters | Type | Description
+---------- | ---- | -----------
+data_set_id (required) | `string` | UUID for accessing the data set
+version (required) | `string` | Version identifier, eg: `en-0`
+result_format | `string` | Query response format: `csv`, `json`, or `geojson` (default is `json`)
+geometry_format | `string` | Either `wkt` or `geojson` for all geometry values (default is `geojson`)
+geojson_feature_key | `string` | Property name to use as geometry when rendering `geojson`
+limit | `integer` | Number of rows to return - the default value is also the maximum: 250 (see <a href="#pagination">Pagination</a>)
+offset | `integer` | Results will be returned starting at the row number specified (see <a href="#pagination">Pagination</a>)
+select | `string` | Comma-separated list of column names to return
+order | `string` | Specify the order of the returned results (see <a href="#ordering">Ordering</a>)
+where | `string` | Conditions for performing query (see <a href="#conditions">Conditions</a>)
+
+<!-- ## Response and Results -->
+
+## Pagination
+
+Each query response is limited to 250 results. To view entire response either use the `export` endpoint to render the results of the query, or use `limit` and `offset` arguments to paginate over results, until no more values are found. 
+
+## Ordering
+
+> ...&order=p0 ASC
+
+Pass in either `ASC` or `DESC` after specifying a column to see results in ascending or descending order, respectively.
+
+## Conditions
+
+The `where` argument supports a number of comparison operators and geospatial functions:
+
+Symbol | Alias | Description | Use
+------ | ----- | ----------- | ---
```

Comment:
Is there any way to put these examples in the cURL column?

Reply:
Not the way I want to and I'm assuming you're imagining the same thing where the right column is an extension of the table. The only way I can see how to do it is to number them and do them like the other examples I've done elsewhere.